.gitlab-ci.yml

```diff
@@ -4,7 +4,8 @@ stages:
     - deploy
 
 variables:
-    ARTIFACT_PATH: eva-accession-ws/target/eva-$ENVIRONMENT_NAME.war
+    ACC_WS_ARTIFACT_PATH: eva-accession-ws/target/eva-acc-ws-$ENVIRONMENT_NAME.war
+    ACC_CLUST_ARTIFACT_PATH: eva-accession-clustering/target/eva-acc-clust-$ENVIRONMENT_NAME.jar
```

Comment:
I would prefer to keep the variable names with complete words. We could use `WS_ARTIFACT_PATH` and `CLUSTERING_ARTIFACT_PATH` if you don't want them to be that long

Reply:
Done. Although one could argue that `WS_ARTIFACT_PATH` is a contraction as well.